.prettierignore

```diff
@@ -1 +1,2 @@
 cookieJar.js
+*.html
```

Comment:
I prefer we keep prettier running on HTML files. I'm guessing the concern here was that it's prettifying our minified Alloy base code and Target prehiding snippet. In other places, we've used a `<!-- prettier-ignore -->` HTML comment to tell prettier to ignore those script tags. I'll make this update.